includes/bootstrap.php

```diff
@@ -260,6 +205,7 @@ function() {
 			apply_filters( 'dt_network_site_connection_enabled', true )
 		) {
 			\Distributor\Connections::factory()->register( '\Distributor\InternalConnections\NetworkSiteConnection' );
+			load_plugin_textdomain( 'distributor', false, basename( dirname( DT_PLUGIN_FULL_FILE ) ) . '/lang' );
 		}
```

Comment:
```suggestion
		}
		load_plugin_textdomain( 'distributor', false, basename( dirname( DT_PLUGIN_FULL_FILE ) ) . '/lang' );
```

Ensures the translations load for sites using only external connections.